src/challenges/challenge_data.cpp

```diff
@@ -79,7 +80,46 @@ ChallengeData::ChallengeData(const std::string& filename)
         return;
     }
 
+    const XMLNode* unlock_list_node = root->getNode("unlock_list");
+    if (unlock_list_node == NULL)
```

Comment:
There could be a default to avoid having to specify that everywhere

Reply:
Default handling added, will push when other changes are ready. The node isn't necessary anymore (though I won't remove it from the challenge file unless I have another reason to update them)